src/component.js

```diff
@@ -136,7 +136,7 @@ const TinyRendererComponentMixin = {
     return this.node;
   },
 
-  receiveComponent() {
+  receiveComponent(nextElement, transaction, context) {
```

Comment:
I had neglected this since I wasn’t using them in this file. I suppose it’s worth adding them, regardless, though?